.github/actions/spell-check/expect.txt

```diff
@@ -1338,6 +1347,7 @@ modulekey
 MONITORINFO
 MONITORINFOEX
 MONITORINFOEXW
+monitorinfof
```

Comment:
`monitorinfof` was already present at line 1352, was it not found by the spellchecker?